docs/reference/release-notes/v2.28/index.md

```diff
@@ -0,0 +1,98 @@
+---
+date: "2020-12-01"
+version: "2.28.0"
+---
+
+# [v2.28](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0) (December 2020 #1)
+
+---
+
+Welcome to `gatsby@2.28.0` release (December 2020 #1).
+
+Key highlights of this release:
+
+- [New `gatsby new`](#new-gatsby-new) - new, interactive way to create a Gatsby site
+- [Improved `react-refresh` integration](#improved-react-refresh-integration)
+- [Feature flags in gatsby-config](#feature-flags-in-gatsby-config) - set your feature toggles without environment variables
+- [Experimental: Lazy images in develop](#experimental-lazy-images-in-develop) - run image transformations only when they are needed by browser
+- [Removed experimental lazy page bundling](#removed-experimental-lazy-page-bundling)
+- [Notable bugfixes](#notable-bugfixes)
+
+Sneak peek to next releases:
+
+- [Less aggressive cache invalidation](#less-aggressive-cache-invalidation)
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](../v2.27/index.md)<br>
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0)
+
+## new `gatsby new`
+
+In previous release we added [interactive way of scaffolding new gatsby project](../v2.27/index.md#create-gatsby) (with `npm init gatsby`). Now we adjusted `gatsby new` command when no arguments are passed to also use the same flow.
+
+## Improved `react-refresh` integration
+
+Gatsby had some support for `react-refresh` (or Fast Refresh as it can be referred to) already. This release improves it and adds better error overlays to be able to fix runtime and compilation error faster!
+
+To use it:
+
+```shell
+GATSBY_HOT_LOADER=fast-refresh gatsby develop
```

Comment:
This is strange to recommend environment variables after we critique them in the next section.

Reply:
It's ... not done yet / supported by flags. I will try to squeze adding fast-refresh toggle through flags before the release, because ... yeah - this is jarring.

Reply:
kinda done here — https://github.com/gatsbyjs/gatsby/pull/28387/files#diff-33a57bbff99725bd550ef514385fe187def17cff7f8734c25c3ee15b945fe334R118-R125 — needs the rest of it wired up so probably this should just happen in its own PR

Reply:
we added flag support ( https://github.com/gatsbyjs/gatsby/pull/28409 ) and adjusted instructions in https://github.com/gatsbyjs/gatsby/pull/28380/commits/a16670a1a801d29207f188b5916f856236351eae